### PR TITLE
feat(ANIMETV7): add presence

### PR DESCRIPTION
**Pull request by Mathe2005**
request to add website to store.

### CODE REVIEW

**Comment by DarkVillager** [comment]
> request to add website to store.

Please follow the required format.

File: websites/A/ANIMETV7/metadata.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+		"id": "1105891230191206432",
+		"name": "ep_1073"
+	},
+	"service": "ANIMETV7",
+	"description": {
+		"en": "ANIMETV7 is a free Anime and Manga streaming website with AniList integration."
+	},
+	"url": "animetv7.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/CtyAaoq.jpg",
```
Comment (DarkVillager) [inline websites/A/ANIMETV7/metadata.json:13]: Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).

File: websites/A/ANIMETV7/presence.ts [hunk 2]
```diff
@@ -0,0 +1,188 @@
+const presence = new Presence({
+		clientId: "1106959657735561256",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+const enum Assets {
+	Logo = "https://i.imgur.com/CtyAaoq.jpg",
```
Comment (DarkVillager) [inline websites/A/ANIMETV7/presence.ts:7]: Same here.
Logo's must be 512x512 as per [our guideliness](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo).

**Review by DarkVillager — CHANGES_REQUESTED** [review]

File: websites/A/ANIMETV7/presence.ts (hunk 2 above)
Comment (Mathe2005) [reply websites/A/ANIMETV7/presence.ts]: check now

File: websites/A/ANIMETV7/metadata.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+		"id": "1105891230191206432",
+		"name": "ep_1073"
+	},
+	"service": "ANIMETV7",
+	"description": {
+		"en": "ANIMETV7 is a free Anime and Manga streaming website with AniList integration."
+	},
+	"url": "animetv7.com",
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/hwnprna.jpg",
```
Comment (DarkVillager) [inline websites/A/ANIMETV7/metadata.json:13]: Make it transparent :)

Comment (Mathe2005) [reply websites/A/ANIMETV7/metadata.json]: done

**Comment by theusaf** [comment]
Hmm. This appears to be a copy/paste fork of [Moopa](https://premid.app/store/presences/Moopa).

**Comment by Mathe2005** [comment]
Ye i am using same code for my website and presences is same too.

**Review by Bas950 — APPROVED** [review]

**Pull request merged**